src/test/java/wolox/training/controllersTest/BookControllerTest.java

```diff
@@ -0,0 +1,168 @@
+package wolox.training.controllersTest;
+
```

Comment:
It is considered good practice to add a photo at the end of the PR with all the tests running satisfactorily.

Reply:
I added photo